service/src/main/java/uk/nhs/adaptors/gp2gp/common/service/IdGenerationService.java

```diff
@@ -0,0 +1,12 @@
+package uk.nhs.adaptors.gp2gp.common.service;
+
+import java.util.UUID;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class IdGenerationService {
```

Comment:
Can be static

Reply:
It would be  useful if it stayed as a spring service. We can inject extra fields later if we need to.